hardhat.config.ts

```diff
@@ -53,7 +53,7 @@ const config: HardhatUserConfig = {
   },
   starknet: {
     venv: ".venv",
-    network: "l2_testnet",
+    network: process.env.CI ? "l2_testnet" : process.env.STARKNET_NETWORK,
```

Comment:
instead of introducing this new env variable, we can simply set `STARKNET_NETWORK` to `l2_testnet` in the CI